sxg_rs/src/headers.rs

```diff
@@ -146,20 +147,14 @@ impl Headers {
         }
         // Google SXG cache sets the maximum of SXG to be 8 megabytes.
         if let Some(size) = self.0.get("content-length") {
-            if let Ok(size) = size.parse::<u64>() {
-                const MAX_SIZE: u64 = 8_000_000;
-                if size > MAX_SIZE {
+            if let Ok(size) = size.parse::<usize>() {
```

Comment:
Perhaps we cam just completely remove this too since we already checked the actual payload body.

Reply:
Although we already checked the actual payload body before checking this `content-length` header, I'd suggest keep the code here. 
We should modify the TypeScript code in the future is to call this `Headers::validate_as_sxg_payload` first and then call `process_html`, because the headers are usually smaller than the body, so that we can fallback to non-SXG earlier.